src/Core/Managed/Shared/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -202,14 +204,48 @@ protected static void BuildTelemetryProcessorChain(XElement definition, Telemetr
                     builder = builder.Use(current =>
                     {
                         var constructorArgs = new object[] { current };
-                        return (ITelemetryProcessor)LoadInstance(addElement, typeof(ITelemetryProcessor), telemetryConfiguration, constructorArgs, null);
+                        return (ITelemetryProcessor)LoadInstance(addElement, typeof(ITelemetryProcessor), null, constructorArgs, null);
                     });
                 }
             }
 
             builder.Build();
         }
 
+        protected static void LoadTelemetrySinks(XElement definition, TelemetryConfiguration telemetryConfiguration)
```

Comment:
I wonder why we need a special function to read it. Is it to avoid `<Add` under `<Sinks`?